Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
 
 gemspec
 
```

Comment:
Maybe the default `Gemfile` can switch to the latest stable Rails version

gem 'rails', '~> 7.0.0'